<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="rolling_ball">
    <link name="ball_link">
      <pose>0 0 0.6 0 0 0</pose>
      <inertial>
        <mass>0.1</mass> <inertia>
          <ixx>0.000025</ixx> <ixy>0</ixy> <ixz>0</ixz>
          <iyy>0.000025</iyy> <iyz>0</iyz>
          <izz>0.000025</izz>
        </inertia>
      </inertial>
      <collision name="ball_collision">
        <geometry><sphere><radius>0.025</radius></sphere></geometry> <surface>
          <friction>
            <ode>
              <mu>1.0</mu>    <mu2>1.0</mu2>   </ode>
          </friction>
           <contact>
            <ode>
               <kp>1e4</kp> <kd>10</kd>    <min_depth>0.005</min_depth> <max_vel>0.05</max_vel>   </ode>
          </contact>
        </surface>
      </collision>
      <visual name="ball_visual">
        <geometry><sphere><radius>0.025</radius></sphere></geometry>
        <material><script><uri>file://media/materials/scripts/gazebo.material</uri><name>Gazebo/Orange</name></script></material>
      </visual>
    </link>
  </model>
</sdf>
